### DISCUSSION TITLE
How to globally disable external link icon ?

### DISCUSSION THREAD

**Discussion by ykla** [vuejs/vitepress#3913] [Q&A] [answered]
Hi, I've only found https://github.com/vuejs/vitepress/pull/3607 but this looks like it only sets the navigation bar. So How to globally disable external link icon ?

**Comment by brc-dd** [comment, marked as answer]
Add it to themeConfig https://vitepress.dev/reference/default-theme-config#externallinkicon

It isn't enabled by default, so just remove it from your vitepress config.
